src/controllers/account.js

```diff
@@ -0,0 +1,30 @@
+//@ts-check
+import Account from "../models/accounts.js";
+import { checkAccountExists } from "../utils/helpers.js";
+import { StatusCodes } from "http-status-codes";
+import ErrorResponse from "../utils/errors/index.js";
+import asyncHandler from "../middleware/async.js";
+
+const { CONFLICT, CREATED, BAD_REQUEST, OK } = StatusCodes;
+
+export const createAccount = asyncHandler(async (req, res) => {
+    console.log("Creating new account");
+    
+    const data = req.body;
+
+    const accountAvailable = await checkAccountExists(data.account_number, "account_number");
+
+    if (accountAvailable) {
+        throw new ErrorResponse("Account already exists", CONFLICT);
+    }
+
+    let account = new Account({ ...data });
```

Comment:
I don't there the spread is needed here since we are not mutating the data object. Don't change it, it's just something to have in mind.
```suggestion
    let account = new Account(data);
```